internal/server/kong/ws/config/compat/compatibility_table_test.go

```diff
@@ -1148,6 +1149,53 @@ func TestDisableChangeTracking(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
Please also add the test for ensuring that it is not emitted.

Reply:
@mikefero that is this https://github.com/Kong/koko/pull/509/files#diff-4a012eaac7d341b5a8d54daa458e5091b0d3ceb450237361164a17392134ced6R928

Reply:
I forgot to hit submit on a change request (your name was wrong for this test); this test is not emitted, but we also need one to ensure it is emitted. This is separate from the integration tests and requires a unit test too.